subsys/bluetooth/controller/ll_sw/ull_llcp.h

```diff
@@ -152,6 +152,18 @@ uint8_t ull_cp_remote_dle_pending(struct ll_conn *conn);
  */
 uint8_t ull_cp_remote_cpr_pending(struct ll_conn *conn);
 
+/**
+ * @brief Check if a remote connection param reg is expecting an
+ *        anchor point move response.
+ */
+bool ull_cp_remote_cpr_awaiting_apm_response(struct ll_conn *conn);
+
+/**
+ * @brief Repsond to anhor point move of remote connection param
+ *        reg.
+ */
+void ull_cp_remote_cpr_apm_respond(struct ll_conn *conn, uint16_t *offsets);
```

Comment:
I would suggest that the API is changed to

`ull_cp_remote_cpr_apm_reply()`
`ull_cp_remote_cpr_apm_neg_reply()`

such that it resembles other API's that deal with accepting or rejecting procedures.

Reply:
Check